src/current/cockroachcloud/backup-and-restore-overview.md

```diff
@@ -217,4 +217,4 @@ For practical examples of running backup and restore jobs, watch the following v
 
 - Considerations for using [backup](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup#considerations) and [restore](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/restore#considerations).
 - [Backup collections](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/take-full-and-incremental-backups#backup-collections) for details on how CockroachDB stores backups.
-- [Restoring backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/restoring-backups-across-versions) across major versions of CockroachDB.
\ No newline at end of file
+- [Restoring backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/restoring-backups-across-versions) across major versions of CockroachDB.
```

Comment:
Not sure what changed here?

Reply:
my editor added a newline at the end of the file

from what I can tell some other editors remove newline at end of file

at some point we could probably consider setting up a team-wide [editorconfig](https://editorconfig.org) for these sorts of things

I didn't find an issue for that so I filed this: https://cockroachlabs.atlassian.net/browse/DOC-10825

Reply:
Ah cool! Sorry missed the new line! Thanks for doing this!